doc/howto/network_load_balancers.md

```diff
@@ -28,7 +28,7 @@ lxc network load-balancer create <network_name> <listen_address> [configuration_
 ```
 
 Each load balancer is assigned to a network.
-It requires a single external listen address (see {ref}`network-load-balancers-listen-addresses` for more information about which addresses can be load balanced, depending on the network that you are using).
+It requires a single external listen address (see {ref}`network-load-balancers-listen-addresses` for more information about which addresses can be load-balanced).
```

Comment:
We use load balancer (without hyphen).

Reply:
Right, but in this case we use it as a verb. And when creating a verb from a compound noun, it gets a hyphen (air condition but air-conditioned, gift wrap but to gift-wrap, ...)

Reply:
OK I'll defer to your grammar knowledge on that one. Should we use load-balancer throughout then?

Reply:
No, let's stick with load balancer. The load balancer - to load-balance - load-balanced. That's the common rule. :)